.github/PULL_REQUEST_TEMPLATE/marker.md

```diff
@@ -0,0 +1,55 @@
+---
+When adding a new marker, you must do 2 things:
+- Add an image to show alongside the marker. This image must be stored under `public/photos`, in a folder named after your GitHub username
+- Edit the `places.ts` file under `data`, adding an entry with the following shape:
+```ts
+{
+    author: '...', // First and Last name of the people involved
```

Comment:
Clarify that this can be a list, when there's multiple people involved 🙏

Reply:
You're right, my bad.